src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -4349,4 +4350,34 @@ public void refreshPaginator() {
         setFilePaginatorPage(dt.getPage());      
         setRowsPerPage(dt.getRowsToRender());
     }  
+    
+    public void archiveVersion(Long id) {
```

Comment:
Since we removed the UI does it make sense to leave this here? At the very list, we should have a comment explaining that there currently is no UI and this is here for future use.

Reply:
I'll add a comment. QDR is keeping the UI part for internal use, so it's helpful to have the underlying java code included. (Since the only other change required is in the xhtml, it's also nominally something that others could add by editing an xhtml file after deploying.) Also, fwiw - it was significant work to design so that the code could be called from a workflow and the DataPage - keeping this here might  help avoid future edits that would break one or the other...

Reply:
Gotcha. Adding a comment will be enough, then.